### DISCUSSION TITLE
Introduction - Jonas Søvik

### DISCUSSION THREAD

**Discussion by Jonas-Soevik** [life-itself/community#987] [Introductions]
Ahoy surfers of these waters!
I be Jonas, pleased to make your acquaintance.
Jokes aside, I'm excited to start engaging with you people! I've been surfing the spaces of Vervaeke land, Zak Stein, Bildung, folk high schools/microcolleges, Perspectiva, UTOK, and similar et ceteras for a few years now - and I'm pretty hyped up about the mission of Life Itself!
I was fortunate enough to visit the hub in Bergerac this summer for week 1 of the Respond retreat and later found the website (which got me very excited).
I study psychology and have a dream to be part of starting a new folk high school-kinda model that is more fitting for our modern day and age, incorporating conscious development through bildung and wisdom cultivation, and principles of Effective Altruism to look at how to help in the world.
I am very interested in what kind of education, skills, and institutions are needed for our society to level up, and how we can create spaces or programs that really allow individuals and communities to transform powerfully, so we can foster an emergent movement to "steal the culture", as Vervaeke would say.

I have done a lot of volunteer project work with planning and hosting events, networking (& bringing other people together that I think should talk), and finding guests for various podcasts (Vervaeke, Microcollege, Nordic bildung), and I'd say I'm semi deep into these landscapes. I have some experience in facilitating; both circling/AR, breathwork (1-hour version), embodiment, and discussions.
My skills are in networking, public speaking, and event planning

**Connections**
- I have done some work with Lene Andersen, John Vervaeke (& Vervaeke Foundation), and Jacob Hundt from Thoreau Microcollege, and I'm now looking to join you all in this endeavour!
- I have been to a Danish efterskole ["afterschool] (10th grade kind of folk high school), folk high school, and am currently attending Youth Initiative Program (YIP) in Sweden
- And I am also part of Effective Altruism and Learning Planet, where I hold active roles

I look forward to getting to engage more with you all, and I'm happy to know more people fighting the good fight
Cheers, and welcome to the new year (let's make it a good one!)

**Comment by rufuspollock** [comment]
@Jonas-Soevik a warm welcome! Ahoy indeed 🏴‍☠️

Thanks for the great introduction with all the info about your background, interests and commitments.

From what I understood you'd be interested in getting involved in Life Itself is that right? 😄

**Comment by Jonas-Soevik** [comment]
Hey Rufus!

Yeah I've had a few calls with Matthew about what going on and especially
the networking project.
I read a few of your articles and I got really fired up, so I am looking to
see how I might be able to contribute. I've been part of this thought space
for a few years (especially bildung).

I'm friends with Lene Andersen so I thought to ask her about her network -
have you spoken with her about this already?

I'd love to hear more on that you're looking to build at the moment and if
I can do anything.

In May I also have the option to do a 2-4 week internship as part of Youth
Initiative Programs curriculum, and I was thinking of asking Life Itself if
anything like that could be possible. Alternatively I've also thought of
asking Jonathan Rowson at Perspectiva

Best from YIP!

**Reply by rufuspollock** [reply]
> I'm friends with Lene Andersen so I thought to ask her about her network -
have you spoken with her about this already?

Yes, I did a call with her back in May and talked about her work and also about https://developmentalspaces.org. And we planned to interview her for our podcast too.

> I'd love to hear more on that you're looking to build at the moment and if I can do anything. ...

What kind of time capacity do you have?

> In May I also have the option to do a 2-4 week internship as part of Youth
Initiative Programs curriculum, and I was thinking of asking Life Itself if
anything like that could be possible. 

We'd definitely be interested! Drop us a line at hello@lifeitself.org to follow up about that.

**Reply by rufuspollock** [reply]
@Jonas-Soevik just "at-ing" you to ensure you saw my follow up above.

**Comment by Jonas-Soevik** [comment]
Hey Rufus,

Yeah, I did. I've just been overwhelmed this week.

Awesome with Lene, I won't pester her with network questions again then.

To be honest, I'm not sure right now. No more than 2-4 hours per week, as
YIP is pretty full 7-7 days, but I might also get a project job Jan-April
that I don't know how will influence me.

I just started with showing interest in DDS and having a few conversations
and now I'm suddenly here. I'll have to start slow now, but from July I can
invest a lot more.

Also through Matthew I found out my network isn't much different from yours
(life itself+you people in it), so I'm also less confident how much I can
help there.

Dope, will drop a mail there 🤙

**Reply by rufuspollock** [reply]
All sounds good.

**Comment by elisapaka** [comment]
Hey @Jonas-Soevik! I'm a bit late to the party haha, but I just saw your post and recognised you from New Ways of Knowing (where we grappled with Nora Bateson's symmathesy mapping in the breakout room on Wednesday! 😅) I'm at the Bergerac hub and doing an internship at Life Itself at the moment. I'm also involved with Perspectiva. I just want to say welcome, and that I'd love to connect over the interests that you mention, especially the stuff around education. If you'd like to have a chat at any point or have questions about Life Itself, feel free to reach out!

**Reply by Jonas-Soevik** [reply]
I wrote you on Beiner's course platform, but I guess you don't use it ;)
I would absolutely love to chat. I'm actually looking to perhaps do an internship in May with either Life Itself or Perspectiva, so your input would be a blessing in helping me decide 🙏
